packages/strapi-generate-api/templates/bookshelf/service.template

```diff
@@ -1,3 +1,4 @@
+/* global <%= globalID %> */
```

Comment:
Are you sure we should add this comment? We could also update the lint configuration...

Reply:
Okay, it makes sense. We can add it!